migreringer/src/main/resources/db/migration/dvhDS/2.0/V2.0_01__unmap-klage-hjemmel.sql

```diff
@@ -0,0 +1,7 @@
+alter table BEHANDLING_DVH set unused (RELATERT_TIL, ENDRET_AV);
```

Comment:
ENDRET_AV til unused vil ikke gi problemer for kjørende poder?

Reply:
Jo, tar den i neste